google/cloud/bigquery/CMakeLists.txt

```diff
@@ -22,4 +22,7 @@ endif ()
 
 if (experimental-bigquery_rest IN_LIST GOOGLE_CLOUD_CPP_ENABLE)
     include(bigquery_rest.cmake)
+    if (BUILD_TESTING)
```

Comment:
These days we would probably add this unconditionally, and let the directory skip everything if testing is disabled.